datasets/plugins/batch/batch_dataset_plugin.py

```diff
@@ -60,90 +63,223 @@ def _get_path_filters_columns(
             filters = [("run_id", "=", query_run_id)]
         return path, filters, read_columns
 
+    def read(
+        self, columns: Optional[str] = None, run_id: Optional[str] = None, **kwargs
+    ) -> Union[pd.DataFrame, "ps.DataFrame", "dd.DataFrame"]:
+        if DataContainerType.SPARK_PANDAS_OR_PANDAS == BATCH_DEFAULT_CONTAINER:
+            try:
+                return self.read_spark_pandas(colums=columns, run_id=run_id, **kwargs)
+            except ImportError:
+                return self.read_pandas(colums=columns, run_id=run_id, **kwargs)
+        elif DataContainerType.SPARK_PANDAS == BATCH_DEFAULT_CONTAINER:
+            return self.read_spark_pandas(colums=columns, run_id=run_id, **kwargs)
+        elif DataContainerType.PANDAS == BATCH_DEFAULT_CONTAINER:
+            return self.read_pandas(colums=columns, run_id=run_id, **kwargs)
+        elif DataContainerType.DASK == BATCH_DEFAULT_CONTAINER:
+            return self.read_dask(colums=columns, run_id=run_id, **kwargs)
+        else:
+            raise ValueError(f"{BATCH_DEFAULT_CONTAINER} does not exist")
```

Comment:
Please review this and semantic behavior of a default returned DataFrame.

Reply:
want to make sure I am understanding right - BATCH_DEFAULT_CONTAINER is now something the end user must define in each project?